docs/blog/2019-03-22-introducing-gatsby-preview-beta/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Introducing Gatsby Preview Beta
+date: 2019-03-22
+author: Marisa Morby
+excerpt: "Introducing the Gatsby Preview Beta: learn more and sign up today for early access to our first commerical product."
+tags:
+  - preview
+  - cloud
+  - commercial
+---
+
+## What we’ve been up to
+
+Back in July we were excited to send out our [alpha version of Gatsby Preview](https://www.gatsbyjs.org/blog/2018-07-17-announcing-gatsby-preview/). The team has been working hard getting your feedback and incorporating it into Gatsby Preview beta.
+
+We’re excited to let you know that this week, Gatsby Preview beta will be officially opening!
```

Comment:
What does "officially opening" mean in this context? When I got to the end of the post and it talked about a closed beta, I wasn't sure.